src/components/watermark/watermark.css

```diff
@@ -0,0 +1,9 @@
+
+.sprite-image {
+    margin: auto;
+    user-select: none;
+    max-width: 48px;
```

Comment:
One last thing, it seems kind of small to me

Reply:
Hmm, this was the size specified in @carljbowman's design...